docs/SOPs/Managed_access/Pre-submission_SOP.md

```diff
@@ -0,0 +1,70 @@
+
+1. Email Jonathan at jlawson@broadinstitute.org with the [DUOS project information fields](https://docs.google.com/document/d/18pzeKafFQZ0rhqrb2DLOCciHUu4On4g2rtgnqygIfTE/edit#heading=h.674bm0dao5mr), using the information from the DCA agreement. This is mainly information about the dataset, the only personal data shared with the Broad is the PI’s name.
+2. Register in ingest the project’s DUOS id provided by Jonathan
+
+## Send instructions to contributor
+Send an email to the data contributor with instructions on next steps*
+
+Send to the contributor, with the wrangler email in CC
+1. An outline of next steps
+2. The metadata spreadsheet guide
+3. A metadata spreadsheet template
+4. A reminder to not share any sensitive data or metadata via email or other means of communication. 
+
+[Template email](#Email-template--Instructions-for-the-contributor){: .btn .btn-blue }
+
+
+## Prepare a secure storage area
+Prepare a secure storage area for the projects’ data and metadata. 
+1. Create a storage area using the project's uuid as name \
+`$ hca-util create <uuid-of-project>` 
+2. The contributor needs a set of credentials to be able to upload data to the hca-util area. 
+Ask a developer to follow [these instructions](https://ebi-ait.github.io/hca-ebi-wrangler-central/SOPs/Introduction/dataset_wrangling_SOP.html#aws-user-for-contributors) to create the credentials. The only information needed is the contributor’s name and email address. 
+
+
+
+For next steps see [Managed access dataset - Data Transfer SOP](https://ebi-ait.github.io/hca-ebi-wrangler-central/SOPs/Managed_access/Data_Transfer_SOP.html)
+
+### Email template - Instructions for the contributor
+
+> Dear [name/s of contributor/s],
+>
+> Thank you for signing the Data Contributor Agreement.
+> The next steps are:
+> 1. Assemble your metadata to meet the HCA standard
+> Attached to this email you will find a metadata spreadsheet where you can fill in all the details of your experimental metadata. A guide to help in this process can be found here: [Spreadsheet Quick Guide](https://ebi-ait.github.io/hca-metadata-community/contributing/spreadsheet-guide.html)
+>
+> If you’ve used organoid or cell line samples or you’ve performed spatial transcriptomics experiments please let me know so I can send you additional information. It’s possible to customise the spreadsheet to better fit your experiment design, so please get in touch if this is something I can help with.
+> 2. Upload your data and metadata to our secure cloud space
+> I am preparing a secure area specific for your data submission where you can upload your data files and the filled-in spreadsheet. Please let me know once you are happy you have filled out the spreadsheet and are ready to upload your data and metadata files, and I'll send you the details of this upload area and instructions on next steps.
```

Comment:
Comment: just try and make the naming of things consistent. I think we call this 'secure storage area' before. Maybe we should rename it to that here again. 

replace 'for your data submission'  --> for your project, where